apps/internal/services/fixture_factory.py

```
@@ -0,0 +1,276 @@
+from django.utils import timezone
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)